src/hlc.h

```diff
@@ -44,17 +44,18 @@
 #undef STRICT
 
 // disable some warnings triggered by HLC code generator
-
+
 #ifdef HL_VCC
-#	pragma warning(disable:4702) // unreachable code
-#	pragma warning(disable:4100) // unreferenced param
-#	pragma warning(disable:4101) // unreferenced local var
-#	pragma warning(disable:4102) // unreferenced label
-#	pragma warning(disable:4700) // uninitialized local variable used
+#	pragma warning(disable:4702) // unreachable code
+#	pragma warning(disable:4100) // unreferenced param
+#	pragma warning(disable:4101) // unreferenced local var
+#	pragma warning(disable:4102) // unreferenced label
+#	pragma warning(disable:4700) // uninitialized local variable used
 #	pragma warning(disable:4703) // potentially uninitialized local
-#	pragma warning(disable:4723) // potential divide by 0
-#	pragma warning(disable:4715) // control paths must return a value
-#	pragma warning(disable:4716) // must return a value (ends with throw)
+#	pragma warning(disable:4723) // potential divide by 0
+#	pragma warning(disable:4715) // control paths must return a value
+#	pragma warning(disable:4716) // must return a value (ends with throw)
+#	pragma warning(disable:4701) // potentially uninitialized local variable

```

Comment:
what's going on here exactly?

Reply:
This file had mixed line endings. 
We only added `#pragma warning(disable:4701) // potentially uninitialized local variable`